lib/pages/main/main_vc.dart

```diff
@@ -95,362 +49,219 @@ class _MainVCState extends State<MainVC> {
     });
   }
 
+  void getPackageInfo() async {
+    var _packageInfo = await PackageInfo.fromPlatform();
+    setState(() {
+      packageInfo = _packageInfo;
+    });
+  }
+
+  static List<Widget> _widgetOptions = <Widget>[
+    HomePage(),
+    MyReportsPage(),
+    MyReportsPage(),  // TODO: Public map
+    TutorialPage(true),
+    SettingsPage(),  // TODO: Create new settings page
+    SettingsPage(),  // TODO: Create info page
```

Comment:
Those `TODOs` are to be done in separate stacked PRs. I think this way its easier to organize and review. One pull request for each individual page